scripts/dependency/install-android-jar.sh

```diff
@@ -12,14 +12,14 @@
 #
 # Adapted from https://github.com/robolectric/robolectric/blob/master/scripts/install-support-jar.sh
 
-jarLocation="$ANDROID_HOME/platforms/android-19/android.jar"
+jarLocation="$ANDROID_HOME/platforms/android-21/android.jar"
```

Comment:
is this the opportunity that tips the scale and we make these into variables at the top:
android_version=android-21
then use $android_version